mne/report/report.py

```diff
@@ -831,6 +831,7 @@ def __init__(
         collapse=(),
         verbose=None,
     ):
+        self.projs = projs  # Initialize self.projs
```

Comment:
did this line need to move up?  Please avoid purely cosmetic changes, it makes the important changes harder to focus on when viewing the diff.